features/resources/stateproof/v1_GetBlock_txn.json

```diff
@@ -0,0 +1,272 @@
+{
+  "block": {
+    "earn": 20,
+    "fees": "A7NMWS3NT3IUDMLVO26ULGXGIIOUQ3ND2TXSER6EBGRZNOBOUIQXHIBGDE",
+    "frac": 5749999177,
+    "gen": "sandnet-v1",
+    "gh": "Bezj3m9sLM49tsNhGbB2BdZIeCpKHkDoaCEV8aDlR2I=",
+    "prev": "blk-BADIGFOQZL4QSSLK77OQ7E3542LI57LFX44HD62EEY5IPJ4IH3UA",
+    "proto": "future",
+    "rate": 249999999,
+    "rnd": 823,
+    "rwcalr": 500000,
+    "rwd": "7777777777777777777777777777777777777777777777777774MSJUVU",
+    "seed": "ipiaZnbd0ga97aNSaVLdhif+xH1mJ1sSAkTLTlUEu1E=",
+    "spt": {
+      "0": {
+        "n": 1024
+      }
+    },
+    "tc": 2,
+    "ts": 1658944706,
+    "txn": "XuNUTARN+9KDJLDZ0DnWKyFfCd7DJ+qsnQ8pvwJ0XEg=",
+    "txn256": "ME1NR8Z+cqGCYrgcbW8vjFfzt7TKaOraexjP6SiQw0g=",
+    "txns": [
+      {
+        "txn": {
+          "fv": 821,
+          "lv": 1821,
+          "snd": "XM6FEYVJ2XDU2IBH4OT6VZGW75YM63CM4TC6AV6BD3JZXFJUIICYTVB5EU",
+          "sp": {
+            "P": {
+              "hsh": {
```

Comment:
I wonder why the proofs are empty.
I would expect to have at least one

Reply:
These are generated when I start a network with sandbox, do you have a more complex example?

 The SDKs use msgpack for this endpoint so I need to convert this from json to msgp also.

Reply:
Taking a second look here, I'm pretty sure that this represents invalid proof. The depth is 1 and there is no digest in the slice. However, Proofs related to the merkle signature scheme look fine.
I suggest we look at it together